playbooks/run-openstack-ansible-security.yml

```diff
@@ -2,5 +2,18 @@
 - name: os-hardening for all hosts
   hosts: all:!vyatta-*
   any_errors_fatal: true
+
   roles:
     - role: ../../openstack-ansible-security
+
+  tasks:
+    - name: install ntp on ubuntu
```

Comment:
need to add retry and do not need to have 2 separate tasks. You can instead use package.